### DISCUSSION TITLE
Bootstrap.min.css file

### DISCUSSION THREAD

**Discussion by SwampDonkeyDW** [twbs/bootstrap#35330] [Q&A] [answered]
Why is this file all jumbled together and basically unresponsive and the broostrap.css file is setup the way it most likely should be? Is anyone else having this problem?

**Comment by mdo** [comment, marked as answer]
We provide human readable compiled CSS files and then minified CSS files that remove all white space for smaller file sizes. Up to you which you use :).

**Reply by SwampDonkeyDW** [reply]
Thank you for the response. I am just starting with bootstrap and discovered this answer about an hour. I apologize for being that guy lol. Thanks again :)

**Reply by SwampDonkeyDW** [reply]
Thank you for the response. I am just starting with bootstrap and discovered this answer about an hour after doing more research. I apologize for being that guy lol. Thanks again :)
